tests_v2/unit/app/utils/test_gadm.py

```diff
@@ -22,3 +22,39 @@ async def test_extract_level_gid() -> None:
     assert extract_level_id(0, match3) == "IDN"
     assert extract_level_id(1, match3) == "35"
     assert extract_level_id(2, match3) == "4"
+
+
+@pytest.mark.asyncio
+async def test_fix_id_pattern_unknown_gadms_pass_through() -> None:
```

Comment:
Awesome job adding all these tests, @dmannarino ! 💪 
Please consider refining the test descriptions. The test descriptions should describe the code's behavior on some input/context.

To help me, I critique the descriptions I write by how quickly I can identify the problem just by reading the description. Also, I imagine being in a design meeting with the team and somebody asking me what `utils/gadm.py` does. In response, I want to be able to just read the test names. The names are descriptive enough that they don't need explaining.

As an example:
```suggestion
async def test_GADM_ids_are_unchanged_if_they_are_NOT_4_1() -> None:
```
and so on down the list...

I hope this helps.